clients/go/zbc/client_test.go

```diff
@@ -29,20 +29,11 @@ import (
 )
 
 type clientTestSuite struct {
-	suite.Suite
-	envCopy map[string]string
-}
-
-func (suite *clientTestSuite) SetupTest() {
-	suite.envCopy = systemEnv.environCopy()
-}
-
-func (suite *clientTestSuite) TearDownTest() {
-	systemEnv.overwrite(suite.envCopy)
+	*envSuite
 }
 
 func TestClientSuite(t *testing.T) {
-	suite.Run(t, new(clientTestSuite))
+	suite.Run(t, clientTestSuite{envSuite: new(envSuite)})
```

Comment:
What's the different with `new(envSuite)` and not specifying anything? Isn't the zero value of `envSuite` just...a new `envSuite`? Or did I misunderstand how these things work?

Reply:
The zero value of a struct is nil. But instead of new(envSuite), you could have &envSuite{}. There